src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageConsoleWriter.cs

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using NuGet.CommandLine.XPlat.Utility;
+using NuGet.Configuration;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal class ListPackageConsoleWriter : IListPackageReportWriter
+    {
+        #region Command runner calls
```

Comment:
https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md

guideline 16: Do not use regions